Missing `export` CMake command when installing

Currently, the root CMake file for utfcpp create the CMake config files when `UTF8_INSTALL` is set. Based on my (ever incomplete) understanding of CMake, it is missing an `export()` command as follows:
```cmake
if(UTF8_INSTALL)
    if(MSVC)
        set(DEF_INSTALL_CMAKE_DIR CMake)
    else()
        include(GNUInstallDirs) # define CMAKE_INSTALL_*
        set(DEF_INSTALL_CMAKE_DIR ${CMAKE_INSTALL_LIBDIR}/cmake/utf8cpp)
    endif()

    install(DIRECTORY source/ DESTINATION include/utf8cpp)
    install(TARGETS utf8cpp EXPORT utf8cppConfig)
    install(EXPORT utf8cppConfig DESTINATION ${DEF_INSTALL_CMAKE_DIR})
    export(EXPORT utf8cppConfig) # <-------- this is missing
endif()
```

Without it, if I use utfcpp in a library of my own and want to create an install target for my library, with exports, CMake currently complains that
```
CMake Error in CMakeLists.txt:
  export called with target "mylib" which requires target "utfcpp" that is not
  in any export set.
```

This error is fixed by adding the `export()` command. I will create a PR to suggest the modification.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,6 +29,7 @@ if(UTF8_INSTALL)
     install(DIRECTORY source/ DESTINATION include/utf8cpp)
     install(TARGETS utf8cpp EXPORT utf8cppConfig)
     install(EXPORT utf8cppConfig DESTINATION ${DEF_INSTALL_CMAKE_DIR})
+    export(EXPORT utf8cppConfig)
 endif()
 
 if(UTF8_SAMPLES)

